devel/create_distro_tarball_debian.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/sh
+#
+# Copyright 2011 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: morlovich@google.com (Maksim Orlovich)
+#
+# The usual mechanism used to develop mod_pagespeed and build binaries is based
+# on merging all dependencies into a single source tree.  This script enables a
+# standard  untar/configure/make flow that does not bundle external libraries.
+# It generates the tarball including the configure (or rather generate.sh)
+# script.
+#
+# This is a variant that's extra-lean, as it targets Debian, which has a
+# particularly extensive package repository. Its file list is based on
+# version 1.11.33.4 (which is partly why it's a separate script and not just
+# a flag).
+
```

Comment:
I agree in principle. The main reason it's not done is there isn't any way of testing the equivalent of this for branch 34 (since that requires a lot of work wrt to gRPC and the like), not is there a working version of the other script for branch 33 around (though that one may be worth doing and is fairly straightforward to test, should I? --- that would provide a sensible way of merging it back into 34, too).